### DISCUSSION TITLE
Remove the IronFist

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#128] [Ideas]
It's strange that the UEF has multiple T3.5 units. The Juggernaut is the more fitting unit since UEF has always been the Tank/caterpillar faction, versus Aeon or Cybran that focuses more on bots and walkers.

Furthermore UEF already has the Titan (cheapest), Percival (more expensive, slow fire, packs a punch), as well as the Pegasus (does decent DPS, and is more of a supporting unit with the anti-tac on the back). They don't need to have the IronFist on top of that.

**Comment by PurpleMossClump** [comment]
With the Juggernaut getting buffed to fill the role of a T3.5 supertank, I think the Ironfist would find itself as a redundant unit on the roster. 

Although, I disagree with the assessment that the UEF is not a walker faction. Mech Marines, Mongoose, Titan, Percival. I think its more that the UEF likes bipedal walkers, while the Cybran leans more into insectoid style walkers.
